### ISSUE TITLE
loss value

### ISSUE THREAD

**Issue by wlwpatriotic** [PKU-ICST-MIPL/Finedefics_ICLR2025#11]
hello，could you please tell me what the loss value is after running the "train.sh" script and completing the training process?

**Comment by Shualite** [comment]
{'loss': 43.4035, 'grad_norm': nan, 'learning_rate': 0.0, 'epoch': 0.01} 
I found the grad_norm is Nan at the beginning of training.
